addons/mail/static/tests/discuss/core/web/command_palette_tests.js

```diff
@@ -45,3 +50,62 @@ QUnit.test("open the chatWindow of a channel from the command palette", async (a
     assert.containsOnce($, ".o-mail-ChatWindow");
     assert.containsOnce($, ".o-mail-ChatWindow-name:contains(general)");
 });
+
+QUnit.test(
+    "List of channels and DMs where the user was tagged in the command palette",
+    async (assert) => {
+        const pyEnv = await startServer();
+        const partnerId = pyEnv["res.partner"].create({ name: "Mario" });
+        const channelId = pyEnv["discuss.channel"].create({
+            channel_member_ids: [
+                Command.create({ partner_id: pyEnv.currentPartnerId }),
+                Command.create({ partner_id: partnerId }),
+            ],
+            channel_type: "group",
+        });
+        const messageId = pyEnv["mail.message"].create({
+            author_id: partnerId,
+            model: "discuss.channel",
+            res_id: channelId,
+            body: "@Mitchell Admin",
+            needaction: true,
+        });
+        pyEnv["mail.notification"].create([
+            {
+                mail_message_id: messageId,
+                notification_type: "inbox",
+                res_partner_id: pyEnv.currentPartnerId,
+            },
+        ]);
+        const { advanceTime } = await start({ hasTimeControl: true });
+        triggerHotkey("control+k");
+        await nextTick();
+        // Switch to partners
+        await editSearchBar("@");
+        await afterNextRender(() => advanceTime(commandSetupRegistry.get("@").debounceDelay));
+        assert.containsOnce($, ".o_command_category_name:contains('Mentions')");
+        assert.containsN($, ".o_command_category", 2);
+        assert.containsOnce($(".o_command_category")[0], ".o_command");
+
+        await click(".o_command.focused");
+        assert.containsOnce($, ".o-mail-ChatWindow");
+        assert.containsOnce($, ".o-mail-ChatWindow-name:contains('Mitchell Admin and Mario')");
+    }
+);
+
+QUnit.test("List of recent three channels and group DMs in the command palette", async (assert) => {
+    const pyEnv = await startServer();
+    pyEnv["discuss.channel"].create({ name: "channel_1" });
+    pyEnv["discuss.channel"].create({ name: "channel_2" });
+    pyEnv["discuss.channel"].create({ name: "channel_3" });
+    pyEnv["discuss.channel"].create({ name: "channel_4" });
+    const { advanceTime } = await start({ hasTimeControl: true });
+    triggerHotkey("control+k");
+    await nextTick();
+    // Switch to channels
+    await editSearchBar("#");
+    await afterNextRender(() => advanceTime(commandSetupRegistry.get("#").debounceDelay));
+    assert.containsOnce($, ".o_command_category_name:contains('Recent')");
```

Comment:
Comments `// Switch to channels/partners` is not that useful. I would just drop them